develop/dev-guide-choose-driver-or-orm.md

```diff
@@ -11,13 +11,12 @@ summary: Learn how to choose a driver or ORM framework to connect to TiDB.
 >
 > - **Full**: indicates that using this driver or ORM does not have any known issues.
 > - **Verified**: indicates that using this driver or ORM might get errors because of compatibility differences between TiDB and MySQL.
-
-<CustomContent platform="tidb">
-
+>
+> <CustomContent platform="tidb">
 >
 > For more information, refer to [Third-Party Tools Supported by TiDB](/develop/dev-guide-third-party-support.md).
```

Comment:
After confirmation, this doc is now applicable to TiDB Cloud. We can remove this CustomContent tag for this doc now. Sorry, I forgot to do that.
https://docs.pingcap.com/tidbcloud/dev-guide-third-party-support